### PR TITLE
fix Super

### PR DESCRIPTION
**Pull request by michaelficarra**

### CODE REVIEW

**Comment by gibson042** [comment]
:+1: for use of CallExpression and MemberExpression. I'm on the fence about `type: "Super"`, but removing its inheritance from Expression is still an improvement.

File: es6.md [hunk 1]
```diff
@@ -31,9 +31,17 @@ extend interface VariableDeclaration {
 # Expressions
 
 ```js
-interface Super <: Expression {
+interface Super {
```
Comment (gibson042) [inline es6.md]: `<: Node`

**Comment by RReverser** [comment]
Agree, should inherit from `Node` at least. Other than that, looks good.

**Comment by michaelficarra** [comment]
Updated. Thanks for the review.

**Comment by ikarienator** [comment]
LGTM :+1: Thanks!

**Comment by RReverser** [comment]
LGTM

**Comment by mikesherov** [comment]
LGTM. SM has said they'll follow us here, so this should be good to merge.

**Comment by RReverser** [comment]
Ok, someone has to do it then :)

**Pull request merged**